src/Notepads.Controls/SetsView/SetsView.cs

```diff
@@ -223,7 +223,17 @@ private void SetsView_SelectionChanged(object sender, SelectionChangedEventArgs
 
         private void UpdateSetSeparators()
         {
-            for (int i = 0; i < Items?.Count; i++)
+            int i = 0;
+            if (SelectedIndex != i)
```

Comment:
use != 0 here. "!= i" might have some side effect in the future. Just move int i = 0 into the for loop and use "0" for the first if-else block.

Reply:
Made changes.